src/tests/receiving/importReceivingTemplate.test.ts

```diff
@@ -0,0 +1,215 @@
+      mainProductService,
+    }) => {
+      const supplierLocation = await supplierLocationService.getLocation();
+      const PRODUCT_ONE = await mainProductService.getProduct();
+
+      STOCK_MOVEMENT = await stockMovementService.createInbound({
+        originId: supplierLocation.id,
+      });
+
+      await stockMovementService.addItemsToInboundStockMovement(
+        STOCK_MOVEMENT.id,
+        [
+          {
+            productId: PRODUCT_ONE.id,
+            quantity: 20,
+            lotNumber: lot,
+            expirationDate: getDateByOffset(new Date(), 3),
+          },
+        ]
+      );
+
+      await stockMovementService.sendInboundStockMovement(STOCK_MOVEMENT.id, {
+        shipmentType: ShipmentType.AIR,
+      });
+    }
+  );
+
+  test.afterEach(async ({ stockMovementShowPage, stockMovementService }) => {
+    await stockMovementShowPage.goToPage(STOCK_MOVEMENT.id);
+    const isButtonVisible =
+      await stockMovementShowPage.rollbackLastReceiptButton.isVisible();
+
+    // due to failed test, shipment might not be received which will not show the button
+    if (isButtonVisible) {
+      await stockMovementShowPage.rollbackLastReceiptButton.click();
+    }
+
+    await stockMovementShowPage.rollbackButton.click();
+
+    await stockMovementService.deleteStockMovement(STOCK_MOVEMENT.id);
+
+    for (const workbook of workbooks) {
+      workbook.delete();
+    }
+  });
+
+  test('Receive shipment using import receiving template', async ({
+    stockMovementShowPage,
+    receivingPage,
+  }) => {
+    let filePath: string;
+    let downloadedExportTemplateFile: WorkbookUtils;
+
+    await test.step('Go to stock movement show page', async () => {
+      await stockMovementShowPage.goToPage(STOCK_MOVEMENT.id);
+      await stockMovementShowPage.isLoaded();
+    });
+
+    await test.step('Go to receiving page', async () => {
+      await stockMovementShowPage.receiveButton.click();
+      await receivingPage.receivingStep.isLoaded();
+    });
+
+    await test.step('Download export template', async () => {
+      const { fullFilePath } =
+        await receivingPage.receivingStep.downloadExportTemplate();
+      filePath = fullFilePath;
+    });
+
+    let parsedDocumentData: unknown[][];
+
+    await test.step('Read downloaded file', async () => {
+      downloadedExportTemplateFile = WorkbookUtils.read(filePath);
+      workbooks.push(downloadedExportTemplateFile);
+    });
+
+    await test.step('Parse csv document to json', async () => {
+      parsedDocumentData = downloadedExportTemplateFile.sheetToJSON();
+    });
+
+    const data: unknown[][] = [];
+
+    await test.step('Parse downloaded template document', async () => {
+      data.push(downloadedExportTemplateFile.getHeaders());
+    });
+
+    await test.step('Input receiving now qty into template', async () => {
+      const documentRow = parsedDocumentData[1];
+      const row = [...documentRow];
+      (row[11] = '20'), (row[12] = 'e2e-comment');
```

Comment:
Can we add consts holding these column numbers that would describe what is behind it?